tests/hpc/slurm_master.pm

```diff
@@ -267,12 +266,29 @@ sub t01_accounting() {
     script_run('sacctmgr show associations');
     record_info('INFO', script_run('sacctmgr show account'));
 
-    script_run("srun --uid=$users{user_1} --account=UNI_X_Math -w slave-node00,slave-node01 date");
-    script_run("srun --uid=$users{user_2} --account=UNI_X_IT -N 2 hostname");
-
-    script_run("srun --uid=$users{user_3} --account=UNI_Y_Biology -N 3 date");
-    script_run("srun --uid=$users{user_4} --account=UNI_Y_Physics -N 3 hostname");
+    my $current_user = $testapi::username;
+    $testapi::username = $users{user_1};
+    my $prompt = $testapi::username . '@' . get_required_var('HOSTNAME') . ':~> ';
+    record_info "$testapi::username", "ok";
+    select_user_serial_terminal($prompt);
+    script_run("srun --account=UNI_X_Math -w slave-node00,slave-node01 date");
+    type_string("su - $users{user_2}", lf => 1);
```

Comment:
I wonder if it helped you to have more user serial terminals. Because with `su -` it does not matter whether the previous account was `root` or non-root UID.

Reply:
i think it would help. But i dont know how to add this functionality

Reply:
`VIRTIO_CONSOLE_NUM=N` could be the way. `VIRTIO_CONSOLE_NUM=2`  creates one root serial console. `VIRTIO_CONSOLE_NUM=3` creates 2 root serial consoles. Maybe I could add the same functionality for user serial consoles. But they would login as `bernhart`, thus you would still need to use `su -`.